packages/11ty/_plugins/figures/annotation/index.js

```diff
@@ -75,7 +75,8 @@ export default class Annotation {
     const uri = () => {
       switch (true) {
         case isImageService:
-          return info()
+          // NB: Annotations for imageServices are *max jpeg*s not the service endpoint
```

Comment:
@cbutcosk in addition to this comment it would kind to our future selves to add a `_plugins/figures/annotations/README.md` file with details about this.